source/_integrations/vitrea.markdown

```diff
@@ -0,0 +1,110 @@
+---
+title: Vitrea
+description: Instructions on how to integrate Vitrea home automation devices with Home Assistant.
+ha_category:
+  - Cover
+  - Switch
+  - Number
+ha_release: 2025.9
+ha_iot_class: Local Push
+ha_config_flow: true
+ha_codeowners:
+  - "@solangek"
+ha_domain: vitrea
+ha_platforms:
+  - cover
+  - number
+  - switch
+ha_integration_type: hub
+---
+
+The Vitrea integration allows you to control Vitrea home automation devices through Home Assistant. This integration supports switches, covers (blinds), and timer (boiler) controls for your Vitrea system.
+
+## Prerequisites
+
+- A Vitrea hub connected to your local network
+- The Vitrea hub must be accessible via TCP/IP
+- Default connection uses port 11502
+
+## Configuration
+
+The Vitrea integration can be configured through the Home Assistant user interface.
+
+### Setup via the frontend
+
+1. Go to **Settings** → **Devices & Services**
+2. Click **Add Integration**
+3. Search for "Vitrea" and select it
+4. Enter your Vitrea hub's IP address and port (default: 11502)
+5. Click **Submit**
+
+The integration will automatically discover and add all available devices connected to your Vitrea hub. The process can take a couple minutes depending on the number of devices.
+
+## Supported devices
+
+The Vitrea integration supports the following device types:
+
+### Switches
+
+- Standard on/off switches
+- Timer-enabled switches with configurable duration (0-120 minutes)
+
+### Covers (Blinds)
+
+- Open, close, and stop controls
+- Position control (0-100%)
+- Current position feedback
+
+### Number Entities (Timers)
+
+- Timer controls for switches that support timed operation
+- Range: 0-120 minutes
+- Used to set default timer duration for associated switches
+
+## Services
+
+### Timer Service
+
+The integration provides a `vitrea.set_timer` service for switches with timer functionality:
+
+```yaml
+service: vitrea.set_timer
+target:
+  entity_id: switch.vitrea_switch_01_01
+data:
+  minutes: 30
+```
+
+## Device Management
+
+All Vitrea devices are automatically grouped under their respective hub device in the device registry. The integration supports:
+
+- **Automatic device discovery**: New devices are detected automatically
+- **Real-time updates**: Device states are updated in real-time via the Vitrea hub
+- **Assumed state**: Entities use assumed state for reliable operation
+
+## Troubleshooting
+
+### Connection Issues
+
+- Ensure your Vitrea hub is powered on and connected to the network
```

Comment:
```suggestion
- Ensure your Vitrea hub is powered on and connected to the network.
```
[End sentences with a period](https://learn.microsoft.com/en-us/style-guide/punctuation/periods). Please fix throughout the page.